src/Evolution/Systems/GeneralizedHarmonic/Equations.hpp

```diff
@@ -80,4 +81,56 @@ struct ComputeDuDt {
       const tnsr::A<DataVector, Dim>& normal_spacetime_vector,
       const tnsr::a<DataVector, Dim>& normal_spacetime_one_form);
 };
+
+/*!
+ * \brief Compute the fluxes of the Generalized Harmonic formulation of
+ * Einstein's equations.
+ *
+ * \details The expressions for the fluxes is obtained
+ * from <a href="https://arxiv.org/abs/gr-qc/0512093">
+ * gr-qc/0512093 </a>.
+ * The fluxes for each variable are given by taking the principal part of
+ * equations 35, 36, and 37, and replacing derivatives \f$ \partial_k \f$
+ * with the unit normal \f$ n_k \f$. This gives:
+ *
+ * \f{align*}
+ * F(\psi_{ab}) =& -(1 + \gamma_1) N^k n_k \psi_{ab} \\
+ * F(\Pi_{ab}) =& - N^k n_k \Pi_{ab} + N g^{ki}n_k \Phi_{iab} - \gamma_1
+ * \gamma_2
+ * N^k n_k \psi_{ab} \\
+ * F(\Phi_{iab}) =& - N^k n_k \Phi_{iab} + N n_i \Pi_{ab} - \gamma_1 \gamma_2
+ * N^i \Phi_{iab}
+ * \f}
+ *
+ * where \f$\psi_{ab}\f$ is the spacetime metric, \f$\Pi_{ab}\f$ its conjugate
+ * momentum, \f$ \Phi_{iab} \f$ is the auxiliary field defined as in Phi,
```

Comment:
I am confused by "defined as in Phi" - is this referring to the code? I would use `'phi'` to make it more clear

Reply:
I'm referencing the Tag `Phi` which defined what Phi is relative to the spacetime metric. The goal here was so that it links to Phi in the rendered docs, but you're right it would be clearer if I said `Tags::Phi` or something.

Reply:
Ah, ok that makes more sense :+1: